vespalog/src/main/java/com/yahoo/log/VespaFormat.java

```diff
@@ -193,4 +200,14 @@ public static void formatException (Throwable t, StringBuilder sbuf) {
         sbuf.append(" nesting=").append(depth);
     }
 
+    static String formatThreadProcess(long processId, long threadId) {
+        if (processId == 0) {
+            return "";
```

Comment:
pretty sure it should be "-" to signify "unknown" here

Reply:
Changed the implementation to assume that process id is always present